README.md

```diff
@@ -137,38 +137,14 @@ transceivers, OpenPower-based Microwatt etc.
 
 ## Install dependencies
 
-For a limited number of configurations, the following script can be used
-to install dependencies.  The script `etc/DependencyInstaller.sh` supports
-Centos7, Ubuntu 20.04, Ubuntu 22.04, RHEL 8, Debian 10 and Debian 11. To correctly install the dependencies with the
-script you need to run as root or prepend `sudo` to the commands below.
```

Comment:
The list of supported platforms should be moved to user/BuildLocally

Reply:
Fixed: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/983